genkit-tools/cli/src/commands/init/init-go.ts

```diff
@@ -36,31 +36,57 @@ interface ModelOption {
 /** Path to Genkit sample template. */
 const templatePath = '../../../config/main.go.template';
 
-/** Ollama init call template. Keep indenting to for expected output. */
-const ollamaInit = `ollama.Init(ctx, ollama.Config{
-        ServerAddress: "http://127.0.0.1:11434", 
-        Models: []ollama.ModelDefinition{{Name: "gemma"}},
-    })`;
-
 /** Model to plugin name. */
 const modelOptions: Record<ModelProvider, ModelOption> = {
   googleai: {
     label: 'Google AI',
     package: 'github.com/firebase/genkit/go/plugins/googleai',
-    init: 'googleai.Init(ctx, "")',
-    lookup: 'googleai.Model("gemini-1.5-flash")',
+    init: `// Initialize the Google AI plugin. When you pass an empty string for the
```

Comment:
Did you test the CLI to make sure it outputs decent format here?

Reply:
yes, outputs here: https://gist.github.com/kevinthecheung/bc41261e4e0e9850f120dd1a2e97dbf3